### ISSUE TITLE
Disconnected by VAC: You cannot play on secure servers

### ISSUE THREAD

**Issue by simono25** [Jire/Charlatano#383]
I don't know if it's the cheat that make me disconnect but i got this error today for the first time and I use this cheat since 11 days. I would like to know if there is any possibility that it can cause this type of problem. Thank you in advance!

**Comment by simono25** [comment]
I am not banned but something is fucking with vac anti-cheat

**Comment by Nvthvnz** [comment]
i had that before, maybe its a different program running in the backround. possibly your using something to boost your fps

**Comment by simono25** [comment]
yeah thank you, I reinstalled steam and csgo, now it's working properly

**Comment by AutisticFurry** [comment]
All you needed to do was check game files in the properties menu in steam. Just to let you know for future or anyone else with this problem.

**Comment by simono25** [comment]
nop, what I did is launch the steam in administrator and launch the hack in admin so vac have the privilege and the hack too
